packages/core/src/boxPalette.ts

```diff
@@ -13,6 +13,7 @@ const boxPaletteVars = {
 	backgroundBodyAlt: '--agds-background-body-alt',
 	backgroundShadeAlt: '--agds-background-shade-alt',
 	border: '--agds-border',
+	borderInput: '--agdds-border-input',
```

Comment:
As the input border color changes between light/dark mode, we need to store this in the `boxPalettes` object